### ISSUE TITLE
About linear learning rate

### ISSUE THREAD

**Issue by manisoftwartist** [open-mmlab/mmdetection#6263]
From the page [mmdetection -Train predefined models on standard datasets](https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html#train-predefined-models-on-standard-datasets)
> Important: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16). According to the linear scaling rule, you need to set the learning rate proportional to the batch size if you use different GPUs or images per GPU, e.g., lr=0.01 for 4 GPUs * 2 imgs/gpu and lr=0.08 for 16 GPUs * 4 imgs/gpu.
 
If I have a single GPU and a batch size of 16, I go with lr=0.02?

**Comment by hhaAndroid** [comment]
Yes

**Comment by hhaAndroid** [comment]
@manisoftwartist workers_per_gpu is mean dataloader num_workers

**Comment by burak43** [comment]
Does this rule also apply to AdamW optimizer's default learning rate of 0.0001 (such as in DETR, Swin etc.)? According to https://github.com/open-mmlab/mmdetection/issues/6356, it looks like it does but at the cost of performance drop. Any recommendations would be appreciated.
